templates/SilverStripe/Forms/FormAction.ss

```diff
@@ -1,6 +1,6 @@
 <% if $UseButtonTag %>
 	<button $AttributesHTML>
-		<% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %>
+		<span><% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %></span>
```

Comment:
(Updating comment to be review)

If we set html directly via $ButtonContent, then we shouldn't wrap that in a span.